D3D12Bundles/D3D12Bundles.cs

```diff
@@ -237,79 +225,97 @@ struct Vertex {
 #if DEBUG
             // Enable the debug layer (requires the Graphics Tools "optional feature").
             // NOTE: Enabling the debug layer after device creation will invalidate the active device.
+            // NOTE: Ensure native debugging is enabled to see the debug messages from this native code in the Visual Studio Debug window!
```

Comment:
This burned me again, I forgot this was required when I first did it in 2020 for D3D12HelloWorld.  This being a new project, didn't have this enabled by default, so here's a reminder.